src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -517,6 +527,16 @@ namespace NServiceBus.AzureServiceBus
         Active = 0,
         Passive = 1,
     }
+    public class NamespacesDefinition : System.Collections.Generic.IEnumerable<NServiceBus.AzureServiceBus.NamespaceDefinition>, System.Collections.IEnumerable
```

Comment:
Ok, maybe it's my english fault :) but this object is about defining one or more namespaces. It's not about defining the same namespace several times.

Is it better NamespaceDefinition**s**?